### PR TITLE
Add `#help` command to csi/vbi

### PR DESCRIPTION
**Pull request by genlu**
csi
Fix https://github.com/dotnet/roslyn/issues/5321. Didn't use MEF as @amcasey suggested.

@dotnet/interactive

### CODE REVIEW

File: src/Scripting/Core/Hosting/CommandLine/CommandLineRunner.cs [hunk 1]
```diff
@@ -329,6 +337,19 @@ private static bool IsTaskAwaiter(Type type)
             return false;
         }
 
+        private static bool IsHelpCommand(string text)
+        {
+            const string helpCommand = "#help";
+            return text != null &&
+                   text.TrimStart().TrimEnd() == helpCommand;
```
Comment (cston) [inline src/Scripting/Core/Hosting/CommandLine/CommandLineRunner.cs]: Assert ```text != null```?

Comment (leppie) [inline src/Scripting/Core/Hosting/CommandLine/CommandLineRunner.cs]: Doesn't `Trim()` does exactly this?

**Comment by cston** [comment]
LGTM

**Comment by ManishJayaswal** [comment]
:+1:

**Comment by genlu** [comment]
@dotnet-bot retest this, please

**Pull request merged**